src/context/mod.rs

```diff
@@ -209,4 +209,9 @@ impl Context {
     pub fn create_string(&self, s: &str) -> RedisString {
         RedisString::create(self.ctx, s)
     }
+
+    #[cfg(feature = "experimental-api")]
+    pub fn emit_aof(&self, io: *mut raw::RedisModuleIO, command: &str, args: &[&str]) {
```

Comment:
The public API of `Context` should not use raw pointers like `*mut raw::RedisModuleIO`; its whole purpose it to provide a safe abstraction over them.

I haven't dived deeply into this, but it seems the correct thing here is to create a wrapper struct around the raw type and pass safe Rust references to it (like `Context` itself).